shell/platform/android/io/flutter/embedding/engine/deferredcomponents/PlayStoreDeferredComponentManager.java

```diff
@@ -202,6 +206,24 @@ public PlayStoreDeferredComponentManager(
     sessionIdToLoadingUnitId = new SparseIntArray();
     sessionIdToState = new SparseArray<>();
     nameToSessionId = new HashMap<>();
+
+    loadingUnitIdToModuleNames = new HashMap<>();
+    String rawMappingString = getApplicationInfo().metaData
+        .getString("flutterDeferredComponentsLoadingUnitMapping", null);
```

Comment:
Should we namespace this instead? Like in ApplicationInfoLoader.java

Reply:
Yeah, ill do that!